packages/edit-site/src/components/page-pages/index.js

```diff
@@ -373,7 +350,7 @@ export default function PagePages() {
 		},
 		[ history ]
 	);
-	const actions = usePostActions( onActionPerformed, PAGE_ACTIONS );
+	const actions = usePostActions( onActionPerformed );
```

Comment:
This hook should be receiving the postType, otherwise it will give inconsistent results.

Reply:
Also, I think this hook is misplaced it should be in "core-data" and not in "editor".